src/style.css

```diff
@@ -1 +1,94 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14.06px;
+  color: black;
+  text-transform: uppercase;
```

Comment:
don't transform all text, only that you need

Reply:
done